.storybook/components/BorderRadius.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Text } from "~/components";
+import { useTheme } from "~/theme";
+import { Unstyled } from "@storybook/blocks";
+
+export const BorderRadiusPresentation = () => {
+  const { themeValues: vars } = useTheme();
+
+  return (
+    <Unstyled>
+      <Box
+        paddingY={10}
+        flexDirection="row"
+        display="flex"
+        flexWrap="wrap"
+        rowGap={13}
+        columnGap={10}
+      >
+        {Object.entries(vars.borderRadius).map(([key, entry]) => (
+          <Box
+            key={key}
+            paddingBottom={10}
+            display="flex"
+            alignItems="center"
+            flexWrap="wrap"
+            flexShrink="0"
+            __flexBasis="30%"
+          >
+            <Box width="100%">{key}</Box>
+            <Text variant="caption" __minWidth="100px">
```

Comment:
**suggestion**: what about using here `pre` tag?

This is a small addition but I think it will be a bit more readable:

![CleanShot 2023-04-19 at 13 44 29@2x](https://user-images.githubusercontent.com/9116238/233064820-6e5e1ac4-19c0-4efa-8e97-5238d1112990.jpg)

Reply:
hmm I don't know... I actually changed that intentionally as pre-font looks like terminal input 😄

Reply:
hsla values are mostly used inside the code anyway, so if think `pre` is suitable here